src/main/java/io/kestra/plugin/kestra/executions/Search.java

```diff
@@ -0,0 +1,225 @@
+                      key: value
+                    auth:
+                      username: root@root.com
+                      password: Root!1234
+                    fetchType: STORE # Store the results in a file
+                """
+        ),
+        @Example(
+            title = "Search for execution in success",
+            full = true,
+            code = """
+                id: search_success_executions
+                namespace: company.team.myflow
+                
+                tasks:
+                  - id: search_executions
+                    type: io.kestra.plugin.kestra.executions.Search
+                    kestraUrl: http://localhost:8080
+                    timeRange: PT10H # In the last 10 hours
+                    state:
+                      - SUCCESS
+                    auth:
+                      username: root@root.com
+                      password: Root!1234
+                    fetchType: FETCH # Fetch the results directly in the task output
+                """
+        )
+    }
+)
+public class Search extends AbstractKestraTask implements RunnableTask<FetchOutput> {
+    @Nullable
+    @Schema(title = "If not provided, every pages are fetched",
+        description = "For example, set to 1, it can be used to only fetch the first 10 results used with `size`.")
+    private Property<Integer> page;
+
+    @Nullable
+    @Builder.Default
+    @Schema(title = "The number of namespaces to return per page.")
+    private Property<Integer> size = Property.ofValue(10);
+
+    @Nullable
+    @Builder.Default
+    @Schema(title = "The way the fetched data will be stored.")
+    private Property<FetchType> fetchType = Property.ofValue(FetchType.STORE);
+
+    @Nullable
+    @Schema(title = "The flow scope, if its a USER or a SYSTEM flow scope, being both by default.")
+    private Property<List<FlowScope>> flowScopes;
+
+    @Nullable
+    @Schema(title = "The namespace to filter executions.")
+    private Property<String> namespace;
+
+    @Nullable
+    @Schema(title = "The flow id to filter executions.")
+    private Property<String> flowId;
+
+    @Nullable
+    @Schema(title = "The start date to filter executions.")
+    private Property<ZonedDateTime> startDate;
+
+    @Nullable
+    @Schema(title = "The end date to filter executions.")
+    private Property<ZonedDateTime> endDate;
+
+    @Nullable
+    @Schema(title = "The time range to filter executions.")
+    private Property<Duration> timeRange;
+
+    @Nullable
+    @Schema(title = "A list of execution states to filter by.")
+    private Property<List<StateType>> state;
+
+    @Nullable
+    @Schema(title = "A list of labels to filter executions.")
+    private Property<Map<String, String>> labels;
+
+    @Nullable
+    @Schema(title = "The trigger execution id to filter executions.")
+    private Property<String> triggerExecutionId;
+
+    @Nullable
+    @Schema(title = "Child filter for execution repository interface.")
```

Comment:
```suggestion
    @Schema(title = "To list only child executions of a given flow.")
```

I don't understand this one, what is `childFilter` and how it's meant to work? can you share an example flow?